client/dashboard/app/router/emails.tsx

```diff
@@ -98,14 +79,10 @@ export const chooseDomainRoute = createRoute( {
 	getParentRoute: () => rootRoute,
 	path: 'emails/choose-domain',
 	loader: async () => {
-		// 1) Preload sites
-		const sites = await queryClient.ensureQueryData( sitesQuery() );
-		const managedSites = ( sites ?? [] ).filter( ( site ) => site.capabilities?.manage_options );
+		// Preload all mailboxes
+		const allMailboxes = await queryClient.ensureQueryData( allMailboxesQuery() );
```

Comment:
This query does not seem to be used in the `choose-domain` component? 🤔

Reply:
Removed